docs/developer_docs/front_end_code.rst

```diff
@@ -89,21 +94,27 @@ To create a new Backbone View, you will either add to an existing Javascript fil
 
 ``this.$el.html(this.template(this.model.attributes));`` this line will insert all the attributes of the model into the template for rendering, and then set the HTML of the subsection of the DOM for the view to the resulting HTML.
 
-For this to work, there must be a file ``kalite/coachreports/hbtemplates/reports/hexagon-counting.handlebars`` that contains the Handlebars.js template for this view. For a simple report, the template might look something like this::
+For this to work, there must be a file ``kalite/coachreports/hbtemplates/reports/hexagon-counting.handlebars`` that contains the Handlebars.js template for this view. For a simple report, the template might look something like this:
```

Comment:
I think handlebars templates need to be included in the `kalite/app/static/js/app/` directory, along with other js sources now.